### PR TITLE
Responding to Martin's comments

### PR DESCRIPTION
**Pull request by nthain**
Martin's comments and my responses listed below:

* It would be nice to have a README that points to the rest of the "Tensorflow without a PhD content"
    - Done
* I have mixed feelings about calling this a "codelab" since users are expected to run the code as-is. It is a great annotated sample though.
    - Renamed all files, folders, and in-text references from codelab to tutorial
* goo.gl short links are dead. You have to use the full link now.
   - Are you sure? They still seem to work
* You still have "contributor nthain, you?" at the beginning. You are the only author. You can credit me for the graphics or the slides if you want.
   - Added martin as a contributor for the graphics and slides
* In the first figure, it is not clear what the green boxes and the blue trapezoids are. Adding two arrows with labels would help immensely.
   - I felt the figure was too busy already and so have added more descriptions to the text above
* You should clear all cell outputs before checking the file into GitHub
   - Done
* There is a TODO left in the notebook: **TODO(nthain): Add pre-trained initialization**
   - Removed
* As you said previously, the "virginity" example is probably too much for a public sample
   - Removed
* You removed the "laziness" examples. These were good!
  - I'm not sure I remember what these were. Can you remind me?

- Also added a requirements.txt and instructions for usage.

### CODE REVIEW

File: attention-tutorial/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Attention Based Classification Tutorial
+
+**Recommended time: 30 minutes**
+
+**Contributors: nthain, martin-gorner**
+
+
+This tutorial provides an introduction to building text classification models in tensorflow that use attention to provide insight into how classification decisions are being made. We will build our tensorflow graph following the Embed - Encode - Attend - Predict paradigm introduced by Matthew Honnibal. For more information about this approach, you can refer to:
```
Comment (iislucas) [inline attention-tutorial/README.md]: s/tensorflow/TensorFlow/g

File: attention-tutorial/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+## To Run Locally
+
+1.  Setup a (virtualenv)[https://virtualenvwrapper.readthedocs.io/en/latest/] for
+    the project (recommended, but technically optional).
+    ```
+
+    Python 3:
+
+    ```
+    python3 -m venv env
```
Comment (iislucas) [inline attention-tutorial/README.md]: Add `env` to .gitignore - lets not accidentally have it checked in. :)

File: attention-tutorial/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    ```
+
+    From either to enter your virtual env:
```
Comment (iislucas) [inline attention-tutorial/README.md]: From 'either' what?

Comment (nthain) [reply attention-tutorial/README.md]: Good catch done.

File: attention-tutorial/README.md (hunk 2 above)
Comment (nthain) [reply attention-tutorial/README.md]: Done.

File: attention-tutorial/README.md (hunk 1 above)
Comment (nthain) [reply attention-tutorial/README.md]: Done.
